configuration/configuration_reference.md

```diff
@@ -562,6 +599,85 @@ Two users would be administrators, they are **Jez** and **lqiao**.
 </view>
 ```
 
+
+## &lt;config-repos&gt; {#config-repos}
+
+The `<config-repos>` element is a container of many `<config-repo>`.
+
+### Example
+
+```xml
+<cruise>
+  ...
+  <config-repos>
```

Comment:
Can we make this example use the default config repo behaviour without plugins i.e. xml configuration?
It's not clear (in this section) that these plugins have to be installed separately.